app/controllers/admin/targets_controller.rb

```diff
@@ -39,9 +37,9 @@ def destroy
         redirect_to(admin_conference_targets_path(conference_id: @conference.short_title),
                     notice: 'Target successfully destroyed.')
       else
-        redirect_to(admin_conference_targets_path(conference_id: @conference.short_title),
-                    error: 'Target was successfully destroyed.' \
-                    "#{@target.errors.full_messages.join('. ')}.")
+        redirect_to admin_conference_targets_path(conference_id: @conference.short_title),
+                    flash:  { error: "Could not delete target for #{@conference.title}: "\
+                    "#{@target.errors.full_messages.join('. ')}." }
```

Comment:
By default rails only provides `:notice` and `:alert`. For everything else we need to use the general purpose flash. (Source: [Rails API#Flash](http://guides.rubyonrails.org/action_controller_overview.html#the-flash))